dist/build/INSTALLING.md.tmpl

```diff
@@ -0,0 +1,306 @@
+
+The installation steps that follow are for the release tar or zip file, which uses content from the following repositories:
+
+- the [PostgreSQL database engine source code repository](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish), with changes that provide the protocols, language parsers, and features required by Babelfish. 
+- the [extensions repository](https://github.com/babelfish-for-postgresql/babelfish_extensions) - extensions support the T-SQL protocol, the T-SQL language, the TDS Protocol, and other Babelfish behaviors.
+
+## Prerequisites
+
+### Hardware and Specs
+
+The current installation instructions were tested using `t4g.large`, `t4.large`, and `c6g.xlarge` instances as hosts.
+
+This installation has also been tested on [ami-0fb653ca2d3203ac1 for amd64](https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#Images:visibility=public-images;imageId=ami-0fb653ca2d3203ac1), and [ami-02af65b2d1ebdfafc for arm64](https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#Images:visibility=public-images;imageId=ami-02af65b2d1ebdfafc).
+
+To compile Babelfish, you should have at least 4GB of available memory.
+
+### Required Software
+
+Install the following dependencies:
+
+```
+sudo apt-get update && sudo apt install -y --no-install-recommends \
+  build-essential flex libxml2-dev libxml2-utils \
+  libxslt-dev libssl-dev libreadline-dev zlib1g-dev \
+  libldap2-dev libpam0g-dev gettext uuid uuid-dev \
+  cmake lld apt-utils libossp-uuid-dev gnulib bison \
+  xsltproc icu-devtools libicu66 \
+  libicu-dev gawk \
+  curl openjdk-8-jre openssl \
+  g++ libssl-dev python-dev libpq-dev \
+  pkg-config libutfcpp-dev \
+  gnupg unixodbc-dev net-tools unzip
+```
+
+Many of the Babelfish prerequisites are part of a typical Linux distribution.  You may find that the packages on your distribution use a similar (but not identical) name.  
+
+To build Babelfish, you will need access to a user with root privileges, so you can convey privileges with `sudo`. You'll also need a non-root user to initialize the database; PostgreSQL does not allow a root user to own the `data` directory or start the server.
+
+
+### Set environment variables
+
+Navigate into the extracted distribution folder and set the following environment variables:
+
+```sh
+export JOBS=4 # Adjust to number of cores
+export BABELFISH_HOME=/opt/babelfish/{{VERSION}}
+export PG_CONFIG=${BABELFISH_HOME}/bin/pg_config
+export PG_SRC=$(realpath $PWD)
+```
+
+### Compile ANTLR 4
+
+Unfortunately, there are [no prebuilt C++ binaries for the Antlr 4.9.3 runtime version](https://www.antlr.org/download.html) for Linux. You will need to compile and install ANTLR manually.
+
+First, define the following variables in your environment:
+
+```sh
+export ANTLR4_VERSION=4.9.3
```

Comment:
Can't this script be parameterized to get the ANTLR version externally?

Reply:
This is just the template for the installation instructions, and presumably changing ANTLR versions will require more changes than simply bumping the version number here. Hopefully by the time we get to changing ANTLR versions we've come up with something far less painful for users than this doc.